APE_coordinated_pkg.rst

```diff
@@ -0,0 +1,235 @@
+
+Note, though, that affiliated packages are still considered a part of the
+"Astropy Project".  That umbrella term applies to both coordinated and
+affiliated packages, as well as the overall community.
+
+
+A criteria-based package review process
+=======================================
+
+This APE also proposes a second change to the review process, motivated by the
+concerns in the "Motivation" section above about the committee's review
+process. To counter these concerns, prior to this APE the coordination committee
+has informally adopted a checklist-based approach, checking if certain bullet
+points are met.  While the formal criteria are listed in relatively vague terms
+on the affiliated package web site, the details of how those criteria are met
+have been fairly consistent using an unwritten but roughly standard list. In
+practice, this has meant that affiliated packages are given "provisional" status
+if some of these items are met, or "full" if a larger set of them are.
+
+This de facto checklist implies that a more transparent and understandable
+policy would be to make this checklist formal and completely public, and use it
+directly as the formal review criteria - i.e., a rubric.  These criteria will be
+laid out explicitly in a document (which has already been drafted - see the
+"Implementation" section below), with various levels defined for each criteria,
+generally corresponding to "good", "acceptable", and "unacceptable". This
+will make the review process much faster and (eventually) open the review
+process beyond the coordination committee, thereby address the concerns from the
+"Motivation" section.
+
+
+A community review process for affiliated packages
+==================================================
+
+On a longer term horizon, a criteria-based review checklist will allow the
+review process to change to a more peer-review model.  That is, instead of
+every package being vetted by the coordination committee, the committee can act
+more as editors, sometimes offering the review opportunity to other members of
+the Astropy community.  The details of this process will not be set in stone by
+this APE, as they should be flexible enough to accomodate the experiences of the
+first rounds of reviewers.  But this APE sets some initial steps (see
+"Implementation" below) and makes the general idea the eventual goal for the
+affiliated package review process.
+
+
+Branches and pull requests
+--------------------------
+
+N/A
+
+
+Implementation
+--------------
+
+The changes outlined in this APE will be implemented as three distinct efforts:
+
+* The new package review criteria have already been implemented by the
+  coordination committee and reviewed by the community (Available in Google doc
+  form `here <https://docs.google.com/document/d/15PJf2PROXMa7xwTDvWnjXB_9KNuO2Ia4_kkxJ7MPazE/edit?usp=sharing>`_.
+  These criteria will need to be translated to HTML and put on the affiliated
```

Comment:
HTML may be too specific, we might want to put it in a JSON file (JSON ALL THE THINGS!)

Reply:
I meant that it needs to be stored in some way and *displayed* via HTML (even if it's DOM-modified via JS).  So I clarified that!